server/src/main/java/org/spine3/server/aggregate/AggregatePart.java

```diff
@@ -50,10 +50,26 @@
  */
 public abstract class AggregatePart<I, S extends Message, B extends Message.Builder> extends Aggregate<I, S, B> {
 
+    private final AggregateRoot<I> root;
+
     /**
      * {@inheritDoc}
      */
-    protected AggregatePart(I id) {
+    protected AggregatePart(I id, AggregateRoot<I> root) {
         super(id);
+        this.root = root;
+    }
+
+    /**
+     * Obtains a part state by its class.
+     *
+     * @param partStateClass the class of the state of the part
+     * @param <P>            the type of the part state
+     * @return the state of the part or a default state if the state was not found
+     */
+    protected <P extends Message, A extends AggregatePart<I, P, ?>> P getPartState(
```

Comment:
I expect `IllegalStateException` in that case, but it mentioned in the documentation for the `getRepository` method in the `AggregateRoot` class. Should I repeat it in the documentation for the `getPartState` method in the `AggregatePart` class?